data/docs.yml

```
@@ -37,6 +37,9 @@
       docs:
         - version: Next
           link: /docs/main/next/readme/
+        - version: Latest
+          link: /docs/main/latest/readme/
+          commitId: 066457be3a2c71a99119b2b69652994c8d2d5365
```

772f9a40f54052defc24f8c6b716f9e9b3fe127c this one? 
```suggestion
          commitId: 772f9a40f54052defc24f8c6b716f9e9b3fe127c
```

---

Yes, this is a mistake. Let's see whether this causes the redirect issue.

---

Sadly, the bug is still there.